migrator/index.js

```diff
@@ -142,7 +143,7 @@ async function migrate () {
       prepareSass('app/assets/sass/application.scss'),
       deleteUnusedFiles(filesToDelete),
       deleteUnusedDirectories(directoriesToDelete),
-      upgradeIfUnchanged(filesToUpdateIfUnchanged),
+      upgradeIfUnchanged(filesToUpdateIfUnchanged, '', upgradeIfPossible),
```

Comment:
I'm questioning this interface, the other usage seems to be:

```
upgradeIfUnchanged([layout], layout, additionalStep)
```

That middle parameter is `starterFilePath` which is used as follows:

```
copyFileFromStarter(starterFilePath || filePath, filePath)
```

So I think we can remove the parameter completely replacing the line with:

```
copyFileFromStarter(filePath, filePath)
```

I'm only basing that on reading the code through, does that seem correct to you @BenSurgisonGDS?  If so I suggest we just remove the second parameter.